DATA/production/qc-sync/mft.json

```diff
@@ -82,26 +82,52 @@
         "remoteMachine": "alio2-cr1-qme05.cern.ch",
         "remotePort": "47797",
         "localControl": "odc"
+      },
+      "MFTAsyncTask": {
+        "active": "true",
+        "className" : "o2::quality_control_modules::mft::QcMFTAsyncTask",
+        "moduleName": "QcMFT",
+        "detectorName": "MFT",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource" : {
+          "type" : "direct",
```

Comment:
Is this correct, a dataSource of type "direct" in sync json?

Reply:
Hi @chiarazampolli yes, we have been using a direct data source for this async task (it is essentially track task) in online QC at P2. The reason is that there is only a given fraction of the MUON tracks reconstructed for the purpose of ED and QC. Therefore if we want to reduce the data flow it makes sense to reduce the reconstruction fraction. 

In the future if it is considered convenient to reduce the QC sampling rather than the reconstruction ration we can also introduce sampling in QC.

Reply:
Thanks, fine with me then.